metadata-ingestion/docs/sources/looker/lookml_post.md

```diff
@@ -1,11 +1,27 @@
 #### Configuration Notes
 
-1. If a view contains a liquid template (e.g. `sql_table_name: {{ user_attributes['db']}}.kafka_streaming.events }}`, with `db=ANALYTICS_PROD`), then you will need to specify the values of those variables in the `liquid_variable` config as shown below:
-    ```yml
-      liquid_variable:
-        user_attributes:
-          db: ANALYTICS_PROD
-    ```
+1. If a view contains a liquid template (e.g. `sql_table_name: {{ user_attributes['db']}}.kafka_streaming.events }}`, with `db=ANALYTICS_PROD`), then you will need to specify the values of those variables in the `liquid_variables` config as shown below:
+
+   ```yml
+   liquid_variables:
+     user_attributes:
+       db: ANALYTICS_PROD
+   ```
+
+2. If a view contains a lookml constant (e.g., `sql_table_name: @{db}.kafka_streaming.events;`), its value is resolved in the following order:
+
+- **First, checks the `manifest.lkml` file** for the constant definition.
+  ```manifest.lkml
+    constant: db {
+        value: "ANALYTICS_PROD"
+    }
+  ```
+- **If not found, falls back to `config`**
+
+  ```yml
+  lookml_constants:
+    db: ANALYTICS_PROD
+  ```
```

Comment:
Can you please preview the docs and fix the styling (not sure if you intended to make text bold)

Also add a note why its fine to resolve this only for views. Something like -

Although liquid variables and LookML constants can be used anywhere in lookml code, currently, their values are resolved only for lookml view. This is good enough as DataHub LookML ingestion ingests only views and their upstreams